### DISCUSSION TITLE
Pokétch Apps

### DISCUSSION THREAD

**Discussion by moriel5** [InfiniTimeOrg/InfiniTime#1531] [Ideas]
During my later childhood, I remember having played Pokémon Diamond version, and well, the in-game gadget there was the "Pokétch" (short for Pokémon Watch) that was what caused me to want a smartwatch (back then, even the term "smartphone" didn't exist, much less "smartwatch").

Now, obviously the Pokétch's interface is very bad for a smartwatch (one button to switch between apps (watchfaces, for those who prefer the term) in a linear fashion, with the only way to get to apps that are earlier in the list is to press the button enough times to get to the end, and then more times to start over, something that gets tiring when there are, I believe it got to at least 16), however the different apps you could get in the game (I would expect them to be detailed over at Bulbapedia) could give some inspirations for any developers wanting to build apps for InfiniTime (such as the calculator or calendar apps).

**Comment by TechnologyClassroom** [comment]
- [X] Digital Watch
- [X] Calculator
- [ ] Memo Pad
- [X] Pedometer
- [ ] Pokémon List
- [ ] Friendship Checker
- [ ] Dowsing Machine
- [ ] Berry Searcher
- [ ] Day-Care Checker
- [ ] Pokémon History
- [ ] Counter
- [X] Analog Watch
- [ ] Marking Map
- [ ] Link Searcher
- [X] Coin Toss
- [ ] Move Tester
- [X] Calendar
- [ ] Dot Artist
- [ ] Roulette
- [ ] Trainer Counter
- [X] Kitchen Timer
- [X] Color Changer
- [ ] Matchup Checker
- [X] Stopwatch
- [X] Alarm Clock

https://bulbapedia.bulbagarden.net/wiki/Poketch

Edit: Added Calculator, Coin Toss, and Calendar.

**Comment by dyamon** [comment]
I'm weirdly intrigued by a friendship checker!
*checks watch* Cool, I'm still friends with my buddy Joe!

**Comment by Eve1374** [comment]
Hi, Do you take into account PR / forks in your list ? 
there's calendar : https://github.com/InfiniTimeOrg/InfiniTime/pull/1958
Calculator : https://github.com/InfiniTimeOrg/InfiniTime/pull/1483 (seems to be merged!)
Dice app (coin toss) : https://github.com/InfiniTimeOrg/InfiniTime/pull/1326 (seems to be merged!)

**Reply by TechnologyClassroom** [reply]
Thanks! Updated.

**Comment by jgm240** [comment]
>
